### PR TITLE
Applying the formatter/import order changes after the smallrye parent upgrade

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

File: tck/src/test/java/KeyLocationResolverTest.java [hunk 1]
```diff
@@ -15,8 +15,23 @@
  *   limitations under the License.
  *
  */
-
-
+/*
```
Comment (kenfinnigan) [inline tck/src/test/java/KeyLocationResolverTest.java]: Is the license duplicated here?

File: tck/src/test/java/TestTokenClaimTypes.java [hunk 2]
```diff
@@ -17,19 +17,28 @@
...
-import io.smallrye.jwt.auth.principal.JWTAuthContextInfo;
-import io.smallrye.jwt.auth.principal.JWTCallerPrincipalFactory;
-import org.eclipse.microprofile.jwt.Claims;
-import org.eclipse.microprofile.jwt.JsonWebToken;
-import org.eclipse.microprofile.jwt.tck.util.TokenUtils;
-import org.jboss.arquillian.testng.Arquillian;
-import org.testng.Assert;
-import org.testng.annotations.BeforeClass;
-import org.testng.annotations.Test;
+/*
```
Comment (kenfinnigan) [inline tck/src/test/java/TestTokenClaimTypes.java]: duplicated license?

File: tck/src/test/java/TestTokenWithGroupsPath.java [hunk 3]
```diff
@@ -1,3 +1,22 @@
+/*
```
Comment (kenfinnigan) [inline tck/src/test/java/TestTokenWithGroupsPath.java]: I think there are quite a few duplicate licenses in the files.

Can you check them?

Comment (sberyozkin) [reply tck/src/test/java/TestTokenWithGroupsPath.java]: Sure will do

Comment (sberyozkin) [reply tck/src/test/java/TestTokenWithGroupsPath.java]: @kenfinnigan looks like the default package was causing it so I moved those test classes into a non empty package; also updated all the license headers to have 2019 and starting with a single asterisk

**Pull request merged**